code/_onclick/item_attack.dm

```diff
@@ -81,7 +81,8 @@
 	user.visible_message("<span class='warning'>[user] hits [src] with [I]!</span>",
 		"<span class='warning'>You hit [src] with [I]!</span>")
 	log_combat(user, src, "attacked", I)
-	take_damage(I.force, I.damtype, "melee")
+	var/power = I.force + round(I.force * 0.3 * user.skills.getRating("melee_weapons")) //30% bonus per melee level
```

Comment:
If untrained this will equal 0.

Reply:
The code is force + force * 0.3 * skill, if no skill, no modifier is applied but the base damage is kept

Reply:
So no, the power would just be the force if skill is not present, e.g machete would still be 60 damage with no skill, as it is its base force

Reply:
To clarify @psykzz, the total damage for a machete using this method would be calculated by power = 60(force of the machete) + (60*0.3*skill)

At 0 skill, there would be 0 damage bonus so it would be 60+ 0 aka 60
At 1 skill it would be a 30% increase (18 damage) or 78 damage
At 2 skill it would be 60% increase, bringing it up to 96 damage.

There shouldnt be any case where the total force of the weapon is 0, even with no skill, and if it does occur, investigate the weapons force variable. If its a value higher than 0, it borked somehow, or armor completely negated it

Reply:
All good, this is my bad.